contracts/AddressSet.sol

```diff
@@ -1,10 +1,9 @@
-pragma solidity ^0.4.18;
+pragma solidity 0.4.18;
```

Comment:
Why does this cause a linting problem?

Reply:
warning  Compiler version must be fixed  compiler-fixed

https://consensys.github.io/smart-contract-best-practices/recommendations/
> Contracts should be deployed with the same compiler version and flags that they have been tested the most with. Locking the pragma helps ensure that contracts do not accidentally get deployed using, for example, the latest compiler which may have higher risks of undiscovered bugs. Contracts may also be deployed by others and the pragma indicates the compiler version intended by the original authors.

Reply:
Thanks.